DB_Models_and_Eloquent.md

```diff
@@ -304,6 +304,22 @@ $users = User::all()->groupBy(function($item) {
 ### Never Update the Column
 
 If you have DB column which you want to be set only once and never updated again, you can set that restriction on Eloquent Model, with a mutator:
+
+- In version 9 and above:
+```php
+use Illuminate\Database\Eloquent\Casts\Attribute;
+
+class User extends Model
+{
+    protected function email(): Attribute
+    {
+        return Attribute::make(
+            set: fn ($value) => $this->email ?? $value,
+        );
+    }
+}
+```
+- In older versions:
```

Comment:
```suggestion
- In version 9 and below:
```
This pattern is still valid in L9.  The docs no longer mention it but IIRC Taylor indicated it was not going away anytime soon.  It is not listed in the Upgrade Guide, which is probably a sign that it will be valid through all of L9.